ddtrace/bootstrap/sitecustomize.py

```diff
@@ -74,6 +69,88 @@ def update_patched_modules():
         EXTRA_PATCHED_MODULES[module] = asbool(should_patch)
 
 
+if PY2:
+    _unloaded_modules = []
+
+
+def gevent_is_installed():
```

Comment:
Can we make this a generic helper that takes the module name as argument? Our hope is that `gevent` is the only "bad actor" here, but so that we are ready for new surprises, then we would have the ground ready. Then we could have a global variable with a list of modules for which we want the `auto` option to work, e.g.
~~~ python
should_cleanup = any(is_installed(_) for _ in MODULES_THAT_REQUIRE_CLEANUP)
~~~
How does it sound?